src/features/Dashboard/AssignedClasses/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Col } from '@edx/paragon';
+import { Button } from 'react-paragon-topaz';
+
+import ClassCard from 'features/Dashboard/AssignedClasses/ClassCard';
+
+import 'features/Dashboard/AssignedClasses/index.scss';
+
+const AssignedClasses = () => {
+  const classes = useSelector((state) => state.common.allClasses.data);
+  const [classCards, setClassCards] = useState([]);
+  const numberOfClasses = 3;
+
+  useEffect(() => {
+    // Display only the first 'NumberOfClasses' on the homepage.
+    if (classes.length > numberOfClasses) {
+      setClassCards(classes.slice(0, numberOfClasses));
+    } else {
+      setClassCards(classes);
+    }
```

Comment:
What about:

```suggestion
    setClassCards(classes.length > numberOfClasses ? classes.slice(0, numberOfClasses) : classes);
```

Reply:
It is easier to read with the if and else